src/python/pants/backend/shell/target_types.py

```diff
@@ -346,6 +357,19 @@ class ShellCommandSourcesField(MultipleSourcesField):
     expected_num_files = 0
 
 
+class RunInSandboxSourcesField(MultipleSourcesField):
+    # We solely register this field for codegen to work.
+    alias = "_sources"
+    uses_source_roots = False
+    expected_num_files = 0
+
+
+class RunInSandboxArgumentsField(StringSequenceField):
+    alias = "argv"
```

Comment:
Total bikeshed, but it seems like `args` is used more often than `argv`, e.g. https://www.pantsbuild.org/docs/reference-pytest#args (plus searching the docs for `argv` has many fewer results than `args`)

```suggestion
    alias = "args"
```